src/core/searchUtils.js

```diff
@@ -96,8 +96,11 @@ export function convertFiltersToQueryParams(filters) {
 
 export function convertQueryParamsToFilters(params) {
   return Object.keys(paramsToFilter).reduce((object, key) => {
-    if (typeof params[key] !== 'undefined' && params[key] !== '') {
-      return { ...object, [paramsToFilter[key]]: params[key] };
+    const paramValue = Array.isArray(params[key])
+      ? params[key][0]
```

Comment:
This should match Django's behavior and take the last query param instead of the first. Also, it would be nice if it could log a warning about which params we ignored. Lastly, this could benefit from a comment to say that we're mimicking what the API does (because it's not the right thing to do).

Reply:
As per the discussion in IRC, I am not in favour of mimicking what Django does. I don't think that's actually relevant because we won't ever be sending multiple params to the API, and I think first param is more logical than last param.

I agree that I should add a comment explaining what we're doing here, and I can add logging if you think that's important.

Reply:
My rationale here was that Django is handling the query incorrectly so the frontend should at least mimic Django to make the behavior consistent when the UI converts an HTTP request to an API request.

If we don't want to mimic Django then we should just solve it correctly which would not be to take the first query param. Solving it correctly would be to take all query params and merge them together. For example, `?sort=recommended,users&sort=featured` should become `?sort=recommended,users,featured`

If it's quicker to fix it by taking the first query param then I guess that's fine.